spec/lib/rbac/filterer_spec.rb

```diff
@@ -74,6 +74,12 @@
         expect(results).to match_array [owned_vm]
       end
 
+      it "with :extra_cols finds Service" do
+        FactoryGirl.create :service, :evm_owner => owner_user
+        results = described_class.search(:class => "Service", :extra_cols => [:owned_by_current_user]).first
+        expect(results.first.attributes["owned_by_current_user"]).to be false
```

Comment:
hmm, where are we setting the current user to be something other than the `owner_user`?